### PR TITLE
upgrade metronome to 2.4.0

### PR DESCRIPTION
**Pull request by kensipe**
## High Level Description

What features does this change enable? What bugs does this change fix? High-Level description of how things changed.

## Related Issues

  -[COPS-1814](https://jira.mesosphere.com/browse/COPS-1814) Fixes to cron-utils enables advanced cron expressions.

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [changelong]((https://github.com/dcos/metronome/compare/87976...23fe8ca)
  - [x] Test Results: [link to CI job test results for component](https://travis-ci.org/dcos/metronome/builds/286232887)

### CODE REVIEW

**Comment by kensipe** [comment]
@mesosphere-mergebot bump-ee

File: packages/metronome/buildinfo.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "requires": ["java", "exhibitor"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.com/maven/dcos/metronome/0.2.3/metronome-0.2.3.tgz",
-    "sha1": "ef42cb936a59360ecd0dda3a434f75e1007f3527"
+    "url": "http://downloads.mesosphere.io/metronome/snapshots/metronome-0.2.4.tgz",
```
Comment (ichernetsky) [inline packages/metronome/buildinfo.json]: Just curious why it ended up in the `snapshots` directory?

Comment (kensipe) [reply packages/metronome/buildinfo.json]: this was made to be the same as marathon.   we need to fix our release process.

**Comment by kensipe** [comment]
@mesosphere-mergebot label ready-for-review

**Comment by orsenthil** [comment]
@kensipe  and reviewers: @timcharper and @ichernetsky : I had a comment on the EE part: https://github.com/mesosphere/dcos-enterprise/pull/1549#issuecomment-336555337 

Please address that, and if no changes are required in EE side, please approve that PR.

**Pull request merged**